src/scheduler/job-list.html

```diff
@@ -8,7 +8,7 @@ <h4 class="frame-title">{{'scheduler' | translate}}</h4>
             <button class="icon-filter motech-sidebar-button" type="button">{{'scheduler.filters' | translate}}</button>
         </div>
     </motech-list-header>
-    <motech-list-item ng-if="jobs.rows.length==0" class="alert">No jobs found</motech-list-item>
+    <motech-list-item ng-show="jobs.rows.length==0" class="alert">No jobs found</motech-list-item>
```

Comment:
Why did we chance ng-if to ng-show? — I'm fine with this being ng-show, but from what I know ng-if add/removes an element rather than just showing/hiding it...